services/core-web/src/components/Forms/GenerateDocumentForm.js

```diff
@@ -104,6 +107,16 @@ export const GenerateDocumentForm = (props) => {
             Cancel
           </Button>
         </Popconfirm>
+        <Button
+          className="full-mobile"
+          type="secondary"
+          disabled={props.disabled}
+          onClick={() => {
+            props.preview(props.documentType, Object.assign({}, props.allCurrentValues));
```

Comment:
Why are you using Object.assign here?

Reply:
If I dont create a copy of the form values when I generate a preview it modifies the dates in the actual form and then they dont work